docs/quickstart.rst

```diff
@@ -5,4 +5,568 @@
 Quick Start
 ===========
 
-This chapter will get you started with Borg-Import.
+This chapter will get you started with Borg-Import and covers various use cases.
+
+A step by step example
+----------------------
+
+.. include:: quickstart_example.rst.inc
+
+Archives and repositories
+-------------------------
+
+A *Borg-Import archive* is the result of a single backup (``Borg-Import create``). An archive
```

Comment:
Hmm, did you just copy that from borg and just replace borg by borg-import?

That reads strange at least if not misleading, so that should be checked and changed at all places.

Also, the goal is not to duplicate the borg docs, they can be referred to by linking, but to document borg-import.

So guess this needs to get done before further review.